### PR TITLE
Adding simple animals sounds

### PR DESCRIPTION
**Pull request by Sindorman**
Simple mobs now have 10-20% chance to make sound when talking. This PR adds sounds for: cat, dog, corgi, bats, cow, goat, chicken, chick, spider, parrot, bees.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
as is tradition, you should probably make it toggleable

(something like, toggle mob sounds or something.)

**Comment by Sindorman** [comment]
I do not think that is required. Because it only occurs when they speak which is RNG based and seems to have low chance. Out of 20 minutes surrounded by 3 each animal I only heard them 5 times in total.

For when they are assumed directly I added 50% chance to make sound when they talk.

**Comment by NonQueueingMatt** [comment]
doth not pass the travis, brother

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 1]
```diff
@@ -1,301 +1,306 @@
...
+	mob_size = 4.5//weight based on Chanthangi goats
+	response_help  = "pets"
+	response_disarm = "gently pushes aside"
+	response_harm   = "kicks"
+	faction = "goat"
+	attacktext = "kicked"
+	maxHealth = 40
+	melee_damage_lower = 1
+	melee_damage_upper = 5
+	var/datum/reagents/udder = null
```
Comment (Sindorman) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]:
```suggestion
	udder = null
```

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 2]
```diff
@@ -552,12 +554,36 @@ mob/living/simple_animal/bullet_act(var/obj/item/projectile/Proj)
 		return (0)
 	return 1
 
+/mob/living/simple_animal/verb/make_noise(var/make_sound = TRUE)
```
Comment (Sindorman) [inline code/modules/mob/living/simple_animal/simple_animal.dm]:
```suggestion
/mob/living/simple_animal/proc/make_noise(var/make_sound = TRUE)
```

**Pull request merged**